docs/operating-scylla/admin-tools/sstableloader.rst

```diff
@@ -1,58 +1,76 @@
 SSTableLoader
 =============
 
-Bulk load the sstables from a directory to a Scylla cluster via the **CQL API**.
+Bulk loads SSTables from a directory to a ScyllaDB cluster via the **CQL API**.
 
 .. note::
 
-   This is **different** than Apache Cassandra tool with the same name, which uses internal RPC protocol to load the data.
+   This tool is **different than Apache Cassandra's sstableloader**, which uses an internal RPC protocol to load data.
 
 .. note::
 
-   sstableloader does **not** support loading from encrypted files. For restoring data from encrypted files see :ref:`Migration to Scylla  <cassandra-to-scylla-procedure>`.
+   sstableloader does **not** support loading from encrypted files. For restoring data from encrypted files, see :ref:`Migration to ScyllaDB  <cassandra-to-scylla-procedure>`.
 
+.. note:: 
 
-SSTableLoader can be used to restore data from Scylla or Apache Cassandra backups or to clone data from cluster to cluster.
-It is especially useful, when the number of nodes, or token range distrbution between the source and target cluster are **not** the same. Since CQL API is used, Scylla will take care of distrbution the data.
+    ScyllaDB does not support local counters used by Apache Cassandra 2.0. If you are migrating from Cassandra 2.0 or a later Cassandra version upgraded from 2.0, using SSTableLoader to migrate local counters to ScyllaDB will not work.
```

Comment:
```suggestion
    ScyllaDB does not support local counters used by Apache Cassandra 2.0. 
    You can use SSTableLoader to migrate Counters tables created with Apache Cassandra 2.0, even if Cassandra was updated to a later version.   
```

Reply:
@tzach I'm a bit confused because according to the issue "Using sstableloader to migrate local counters to Scylla will not work".
@vladzcloudius Could you review this PR?

Reply:
Correct, @annastuchlik. One can NOT use `sstableloader` on SSTables containing local counters.

I'm not sure what was the idea of the author of this patch.